code/modules/paperwork/photocopier.dm

```diff
@@ -239,6 +243,71 @@
 	return p
 
 
+
+
+//////Premium(upgraded) photocopier/////
```

Comment:
same as above, codedoc

you can use a more descriptive statement as codedoc instead, for example:

`/// Premium photocopier, a straight upgrade to the standard one`